modules/Bio/EnsEMBL/Compara/Utils/DistributionTag.pm

```diff
@@ -0,0 +1,69 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::Utils::DistributionTag
+
+=head1 DESCRIPTION
+
+Utility methods for writing tag distribution scores
```

Comment:
Is it worth leaving the description more generic for what may be added in the future?
```suggestion
Utility methods for writing distribution tags.
```